exodus_gw/schemas.py

```diff
@@ -35,7 +35,8 @@ class ItemBase(BaseModel):
         "",
         description=(
             "Key of blob to be exposed; should be the SHA256 checksum of a previously uploaded "
-            "piece of content, in lowercase hex-digest form."
+            "piece of content (in lowercase hex-digest form) or the string 'absent' "
+            "(it indicates that content unable to be fetched from the CDN.)"
```

Comment:
I think a little more detail would be good here. Something like...
```suggestion
            "piece of content (in lowercase hex-digest form) or the string 'absent'."
            "An 'absent' object_key allows publishing of items not uploaded."
```

Reply:
Maybe I've misread, but from the Jira tickets in the content deletion epic, it sounds like 'absent' doesn't suggest that the content doesn't exist in the s3 bucket, just that it shouldn't be accessed from some uri. (So, I don't think "allows publishing of items not uploaded" is correct.)

I'd suggest to either remove the explanation for what 'absent' means in the context of exodus-lambda,

```suggestion
            "piece of content (in lowercase hex-digest form) or the string 'absent'."
```
or to maybe fix up the sentence like so:
```suggestion
            "piece of content (in lowercase hex-digest form) or the string 'absent'"
            "(indicates that content shouldn't be fetched from the web_uri)."
```

Reply:
I'll pile on with my own suggestion:

> Key of blob to be exposed; should be the SHA256 checksum of a previously uploaded
> piece of content, in lowercase hex-digest form.
> Alternatively, the string 'absent' to indicate that no content shall be exposed at the given URI.
> Publishing an item with key 'absent' can be used to effectively delete formerly published
> content from the point of view of a CDN consumer.

The main points being:

- don't try to cram too much in a single sentence
- I do think the docs should explain *somewhere* why someone might want to do this.  Hence the last sentence.  I'm borderline whether 'schemas' is really the right place for that though.  The prime alternative would be in the doc string of the update_publish_items function, e.g. right after "Publish items primarily are ...", another sentence mentioning something like "Publish items can also hide previously published items by ..."